wowchemy/layouts/partials/breadcrumb_helper.html

```diff
@@ -0,0 +1,8 @@
+{{ with .Parent }}
+  {{ partial "breadcrumb_helper" . }}
+  <li class="breadcrumb-item">
+    <a href="{{ .Permalink }}">
+      {{ if .IsHome }}{{ i18n "home" }}{{ else }}{{- .Title | emojify -}}{{ end }}
```

Comment:
Change `{{ i18n "home" }}` to `{{ i18n "home" | default "Home" }}` so there's a safe fallback.

Reply:
Done